ghcide/src/Development/IDE/Core/Rules.hs

```diff
@@ -266,40 +280,7 @@ getParsedModuleRule recorder =
 
     -- We still parse with Haddocks whether Opt_Haddock is True or False to collect information
     -- but we no longer need to parse with and without Haddocks separately for above GHC90.
-    res@(_,pmod) <- if Compat.ghcVersion >= Compat.GHC90 then
```

Comment:
good idea, just a thought that we could probably just delete the `Compat.GHC90` enum member entirely and that might point out other places to fix.

Reply:
Agree. I wasn't sure if I can just remove them but it definitely makes sense.